### PR TITLE
Fix #148 help section

### LINKED ISSUE #126: add help section and tutorials

### LINKED ISSUE #148: add FAQ to help section
have it in footer

### PR DESCRIPTION
**Pull request by ManjotS**
fix #126 #148

### CODE REVIEW

File: src/js/help.js [hunk 1]
```diff
@@ -0,0 +1,111 @@
+$(function () {
```
Comment (bogas04) [inline src/js/help.js]: Please no jQuery. You can wrap it in a function and call it.

File: src/js/help.js [hunk 2]
```diff
@@ -0,0 +1,111 @@
...
+      [ 'How do I report a mistake?',
+        'Visit <a href="https://sikhitothemax.org">SikhiToTheMax.org</a> and click "Feedback" ' +
+        'at the bottom of the page.']
+    ]
+  };
+
+  const sidebar = (<div id="sidebar"></div>);
+  const main = (<main></main>);
+  const sidebarList = (<ul></ul>);
+  sidebar.appendChild(sidebarList);
```
Comment (bogas04) [inline src/js/help.js]: You can simply compose all this in JSX. I couldn't complete the entire thing, but this should help you understand how to do it.

```jsx
<div id='help'>
  <div id='sidebar'>
  </div>
  <main>
    {Object.keys(content).map(header => (
      <section id={header} key={header}>
        <h2>{header}</h2>
        <li>
          <a href={`#${header}`}>{header}</a>
          <ul>
            {
              content[header].map(([ firstItem ]) => {
                const title = firstItem.split(' ').join('-').toLowerCase().replace(/[^a-z.\-]/g, '').substr(0, 50);
                return (<li><a href={'#' + title}>{firstItem}</a></li>);
              })
            }
         </ul>
        
        {
           content[header].map(q => {
            const title = q[0].split(' ').join('-').toLowerCase().replace(/[^a-z.\-]/g, '').substr(0, 50);
            return (
              <div class='question' id={title}>
              </div>
            );
          })
        }
      </section>
    )}
  </main>
</div>
```

**Comment by bogas04** [comment]
Sorry for hijacking your PR @ManjotS.
I've updated it so that it uses JSX for composition. Note how the code is almost exactly how it would appear in DOM, this declarative way of writing code helps us reason about it better. I hope you also agree with the same.

Could you please review my changes, if all looks good we can merge!

**Comment by ManjotS** [comment]
lgtm

**Comment by tarunsingh5** [comment]
the help section lgtm

**Pull request merged**